File: C/Assignment6_1.c
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment6_1.c 
// Description : Write a program which accept number from user and print if number ( < 50) small,(> 50 && < 100)medium,(>100)large
// Author      : Aniket Devaba Markad 
// Date        : 16/05/2025
// 
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 

#include<stdio.h>

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 
// 
// Function Name : DisplayNumber
// Description   : This function is used to display number. 
// Input         : int 
// Output        : void
// Author        : Aniket Devaba Markad 
// Date          : 16/05/2025 
// 
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 

void DisplayNumber(int iNo)
{
    if(iNo < 50)
    {
        printf("number is : Small\n");
    }
    else if((iNo >= 50) && (iNo <= 100))
    {
        printf("number is :Medium\n");
    }
    else if(iNo > 100)
    {
        printf("number is : Large\n");
    }
}

int main()
{
    int iValue = 0;
    
    printf("Enter number :");
    scanf("%d ",&iValue);

    DisplayNumber(iValue);

    return 0;
}
//Time Complexcity cant find in this example.

File: Java/Assignment6_1.java
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment6_1.java
// Description : write a program which accept number from user and if number is less than 50 then print small,
//               if number between 50 to 100 then print medium,and if number greater than 100 then print large.
// Author      : Aniket Devaba Markad                        
// Date        : 6/12/2025 
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Number
{

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name : DisplayNumber
// Description   : This function is used to print number based on range  . 
// Author        : Aniket Devaba Markad 
// Date          : 6/12/2025 
// 
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

    public void DisplayNumber(int iNo)
    {

        if(iNo <= 0)
        {
            iNo = -iNo;
        }

        if(iNo > 0 || iNo <= 50)
        {
            System.out.println("Number is : Small");
        }
        else if(iNo > 50 || iNo <= 100)
        {
            System.out.println("Number is :medium");
        }
        else if(iNo > 100)
        {
            System.out.println("Number is :large");
        }
    }
}
public class Assignment6_1
{
    public static void main(String []a)
    {
        int iValue = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("enter the number");
        iValue = sobj.nextInt();

        Number nobj = new Number();

        nobj.DisplayNumber(iValue);

    }
}


